src/app/qgsattributesforminitcode.cpp

```diff
@@ -0,0 +1,84 @@
+#include "qgsattributesforminitcode.h"
+#include "ui_qgsattributesforminitcode.h"
+#include "qgssettings.h"
+
+#include <QFileDialog>
+
+QgsAttributesFormInitCode::QgsAttributesFormInitCode()
+{
+  setupUi( this );
+
+  // Init function stuff
+  mInitCodeSourceComboBox->addItem( tr( "" ) );
```

Comment:
s/tr( "" )/QString()/

tr() is for translation, I guess the empty string is the same for all locales